frontends/web/src/locales/en/app.json

```diff
@@ -692,6 +693,12 @@
         "title": "How to securely pair with your phone"
       }
     },
+    "bitbox02": {
+      "rootFingerprint": {
+        "text": "The root fingerprint is a unique identifier for the wallet that is currently loaded. It can help you distinguish different wallets if you use passphrases.",
```

Comment:
@jadzeidan text sounds good?

Reply:
ping @jadzeidan

Reply:
How about:

"The root fingerprint is a unique identifier for the wallet **currently in use**. It can help you distinguish **between** different wallets if you use passphrases."

Reply:
Makes sense to me